packages/rabbitmq/src/amqp/connection.ts

```diff
@@ -562,11 +562,18 @@ export class AmqpConnection {
     let batchTimer: NodeJS.Timeout;
     let inflightBatchHandler: () => Promise<void>;
 
+    // Normalize batch values but warn consumer about this adjusts
```

Comment:
The logs below are meant to provide further feedback to developers, they should know when we normalize these values for the ease of debugging